yarn.lock

```diff
@@ -10435,11 +10435,6 @@ prr@~1.0.1:
   resolved "https://registry.yarnpkg.com/prr/-/prr-1.0.1.tgz#d3fc114ba06995a45ec6893f484ceb1d78f5f476"
   integrity sha1-0/wRS6BplaRexok/SEzrHXj19HY=
 
-pseudomap@^1.0.2:
-  version "1.0.2"
-  resolved "https://registry.yarnpkg.com/pseudomap/-/pseudomap-1.0.2.tgz#f052a28da70e618917ef0a8ac34c1ae5a68286b3"
-  integrity sha1-8FKijacOYYkX7wqKw0wa5aaChrM=
-

```

Comment:
Hmm..seems interesting. We have had some dependency which was not used by anything as only this changed in the yarn.lock?

Reply:
I tried this again. Copied yarn.lock from develop branch to this one and ran yarn install. And it removes this part again. Seems legit.